packages/apps/docs/src/pages/docs/blogchain/2018/consensus2018-5-lessons-from-your-first-crypto-conference-day-1-2018-04-15.md

```diff
@@ -12,42 +12,36 @@ layout: blog
 
 ### What to do when they bring the lambos to your doorstep
 
-Lesson #1: Prepare for a
-[line to registration](https://twitter.com/mdudas/status/996070779434528769)
-that represents the fundamental problems of throughput and congestion for
-blockchain as a whole.
+Lesson #1: Prepare for a line to registration that represents the fundamental
+problems of throughput and congestion for blockchain as a whole.
 
-TODO: embed
-https://twitter.com/dickerson_des/status/996041484007886848?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E996041484007886848%7Ctwgr%5Ef54a655ebe4bf6e4f65fb28adac49d5e1911205a%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Fcdn.embedly.com%2Fwidgets%2Fmedia.html%3Ftype%3Dtext2Fhtmlkey%3Da19fcc184b9711e1b4764040d3dc5c07schema%3Dtwitterurl%3Dhttps3A%2F%2Ftwitter.com%2Fdickerson_des%2Fstatus%2F996041484007886848image%3Dhttps3A%2F%2Fi.embed.ly%2F1%2Fimage3Furl3Dhttps253A252F252Fpbs.twimg.com252Fprofile_images252F873291993627193344252FpruiILEs_400x400.jpg26key3Da19fcc184b9711e1b4764040d3dc5c07
+TODO: not working
+[line to registration](https://x.com/mdudas/status/996070779434528769)
```

Comment:
looks like this tweet deleted/moved. 

shall I remove it cc @sstraatemans

Reply:
yeah if deleted, lets remove it